src/configurations/adknowledgeportal/style/_style_overrides.scss

```diff
@@ -1,4 +1,4 @@
-@use 'variables' as Portal;
+@use '../../style/variables' as Portal;
```

Comment:
Since we're not using sass load paths anymore, our file references need to be updated. This is easier to reason with IMO.